src/task/ReceiveTURNCredentialsTask.ts

```diff
@@ -38,20 +38,17 @@ export default class ReceiveTURNCredentialsTask extends BaseTask {
     }
 
     const options: RequestInit = {
-      method: 'POST',
+      method: 'GET',
       mode: 'cors',
       cache: 'no-cache',
       credentials: 'omit',
-      headers: {
-        'Content-Type': 'application/json',
-        'X-Chime-Auth-Token':
-          '_aws_wt_session=' + this.joinToken.replace(ContentShareConstants.Modality, ''),
-      },
       redirect: 'follow',
       referrer: 'no-referrer',
-      body: JSON.stringify({ meetingId: this.meetingId }),
     };
 
+    const url: string =
+      this.url +
+      `?m=${this.meetingId}&t=${this.joinToken.replace(ContentShareConstants.Modality, '')}`;
```

Comment:
You can use [Modality.base](https://aws.github.io/amazon-chime-sdk-js/classes/defaultmodality.html#base) for this:

```
const url: string =
      this.url +
      `?m=${this.meetingId}&t=${(new DefaultModality(this.joinToken)).base()}`;
```